phoenix/slackbot/views.py

```diff
@@ -893,8 +906,11 @@ def resolve(self):
         outage.sales_affected_choice = self.dialog_data.get("sales_affected_choice")
         outage.lost_bookings = lost_bookings
         outage.impact_on_turnover = impact_on_turnover
-        solution.save(modified_by=self.actor)
+
+        if not outage.resolved:
+            outage.resolved = True
```

Comment:
Is this like the default?

Reply:
It's there to make sure that when the `Resolve` dialog is being processed the state of `.resolved` will be set to `True`.

Reply:
I see, alright